kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtStatusServiceImpl.java

```diff
@@ -226,6 +226,22 @@ private void fillFromDataService(List<GwtGroupedNVPair> pairs, final String data
                 return Collections.emptyList();
             }
 
+            Collections.sort(gwtNetInterfaceConfigs, new Comparator<GwtNetInterfaceConfig>() {
```

Comment:
should not it be easier with Java 8 Comparators? Just a proposal.

```java
Collections.sort(gwtNetInterfaceConfigs, Comparators.comparing(GwtNetInterfaceConfig::getName));
```

Reply:
As of now Java 8 isn't supported as you might remember. But as we agreed on yesterday's call we will upgrade to GWT 2.8.x and then we can go ahead and use Java 8 APIs.

Reply:
If I'm not wrong, java 8 apis can be used in the server part, not in the client part.
This is server part, so it can be used. https://github.com/eclipse/kura/blob/6eaec61ca692a7bded9db1846eb31570626e175c/kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtComponentServiceImpl.java#L108

Reply:
I agree with @MMaiero. Since it is the server part, we have been using Java 8 and we can also do it here as well.

Reply:
Well I am not sure if it is a good practice to mix Java 7 and 8 targets in a single project. But assuming that we will get GWT 2.8.x anyway, lets move forward to use Java 8 and simple ignore Java 7 from now on.